openwisp_controller/api/serializers.py

```diff
@@ -0,0 +1,137 @@
+import collections
```

Comment:
please move the api directory under `config`, the django app where this logic belongs.

Reply:
Done.